tests/scripts/generate_test_keys.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+"""Module generating EC and RSA keys to be used in test_suite_pk instead of
+generating the required key at run time. This helps speeding up testing."""
+
+import os
+import sys
+from typing import Iterator
+import re
+# pylint: disable=wrong-import-position
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__)) + "/"
+sys.path.append(SCRIPT_DIR + "../../scripts/")
```

Comment:
These three lines are no longer necessary, please remove them. All the dirty stuff is happening inside `scripts_path.py`.

I think `import scripts_path` needs to be above `from mbedtls_dev`.